dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ExceptionUtils.java

```diff
@@ -522,13 +523,21 @@ static String validateProductMatches(String feature, String dependency, List pro
                 applicableProducts.append(InstallUtils.NEWLINE);
                 //installing asset has invalid product version and/or edition
                 if (dependency == null || dependency.isEmpty()) {
-                    if (appliesToVersion.equals("")) //installing asset has invalid product edition only
-                        errMsg = Messages.INSTALL_KERNEL_MESSAGES.getLogMessage(installingAsset ? "ERROR_ASSET_INVALID_PRODUCT_EDITION" : "ERROR_INVALID_PRODUCT_EDITION",
-                                                                                new Object[] { feature, productName, edition, applicableProducts.toString(), productName,
-                                                                                               edition });
-                    else
+                    if (appliesToVersion.equals("")) { //installing asset has invalid product edition only
+
+                        if (((String) params[productEdition]).equalsIgnoreCase("Open_Web") && productName.equalsIgnoreCase("IBM WebSphere Application Server Liberty"))
```

Comment:
Use brackets within the if/else blocks as a best practice, in case other lines need to be added in the future